knative-serving/http-configuration/http1-full-duplex-support.adoc

```diff
@@ -0,0 +1,28 @@
+:_content-type: ASSEMBLY
+include::_attributes/common-attributes.adoc[]
+[id="http1-full-duplex-support"]
+= Full duplex support for HTTP/1
+:context: http1-full-duplex-support
+
+You can enable the HTTP/1 full duplex support for a service by configuring the `features.knative.dev/http-full-duplex` annotation.
+
+NOTE: Verify your HTTP clients before enabling, as older clients may not provide support for HTTP/1 full duplex.
+
+The following example shows how you can use this annotation in a Knative `Service` YAML object:
+
+.Example `KnativeServing` CR that provides full duplex support for HTTP/1
+[source,yaml]
+----
+apiVersion: serving.knative.dev/v1
+kind: Service
+metadata:
+  name: example-service
+  namespace: default
+  annotations:
+    features.knative.dev/http-full-duplex: "Enabled"
```

Comment:
The annotation should be on the revision template, not Service,

this was fixed upstream in https://github.com/knative/docs/pull/5839

Reply:
Also the docs are here: https://knative.dev/docs/serving/services/http-protocol/#http1-full-duplex-support-per-workload